nav_app/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from .forms import GetUserInfoForm
+
+
+def index(request):
+    return HttpResponse("<h1>Welcome to ShareTaxi.</h1>")
+
+
+def book_taxi(request):
```

Comment:
request will come as `json` in `post` body. no need of `form`

Reply:
Done.